src/Hook/useFilterFunctionCombiner.js

```diff
@@ -0,0 +1,25 @@
+import { useFilterReducerContext } from "../context/FilterReducer.context";
+import {
+  getSortedProducts,
+  priceRangeSorter,
+  categoryFilter,
+  brandFilter,
+  outOfStockFilter,
+} from "../reducer/filterReducerFunctions";
+
+export const useFilterFunctionCombiner = () => {
```

Comment:
In PR description. name the file correct. It is misspelled.

Reply:
thanks for pointing it out updated the description.